example/ios/Podfile.lock

```diff
@@ -464,12 +470,20 @@ PODS:
     - glog
   - react-native-create-thumbnail (2.0.0):
     - React-Core
+  - react-native-get-random-values (1.11.0):
+    - React-Core
   - react-native-image-picker (7.1.2):
     - RCT-Folly (= 2021.07.22.00)
     - React-Core
+  - react-native-livestream (2.0.0):
+    - ApiVideoLiveStream (= 1.4.1)
+    - RCT-Folly (= 2021.07.22.00)
+    - React-Core
   - react-native-netinfo (11.3.2):
     - React-Core
-  - react-native-safe-area-context (4.10.1):
```

Comment:
safe area context is not removed. 
still existed in node_modules. 
I guess there was mis-match while running yarn install and pod install. 

Don't need to install again. Just put this back in pod file.